python/compiler/aiecc/main.py

```diff
@@ -1016,6 +1024,9 @@ async def run_flow(self):
                     progress_bar.task,
                     [
                         "aie-opt",
+                        "--aie-materialize-bd-chains",
+                        "--aie-assign-runtime-sequence-bd-ids",
+                        "--aie-dma-tasks-to-npu",
                         "--aie-dma-to-npu",
```

Comment:
use DMA_TO_NPU here?
```python
["aie-opt", f"--pass-pipeline={DMA_TO_NPU ()}" ...
```

Reply:
yeah, looks like we were previously defining it but not using it. weird